meeting-minutes/client-minutes/feb23-client-minutes.md

```diff
@@ -0,0 +1,61 @@
+
+1. We should be able to have multiple admins associated to the same organization.
+2. On our Mentee request cards, for privacy reasons, we should either remove the email address completely from the card, or asterisk out some of the address.
+3. We should ensure that pages with personal information aren't accessible to the public when queried (i.e we shouldn't be able to see a list of Mentee information if we haven't been authenticated). The client screenshared this [github repo](https://github.com/public-apis/public-apis), specifically pointing out the 'Auth' column in the various tables. She notes that this could be too complicated to setup in the remaining time though.
+4. After we finalize the embedding on the Mentee form, we could redirect the Mentee user to our platform. This would be in preparation for the possibility of Mentees having accounts in the future.
+
+Given that there's only 6 weeks left of the course, I mentioned to the client that it may not be feasible to incorporate new functionality, especially since we want to ensure that what we've created is functioning 100% as expected.
+Although it would be great if we could incorporate those things, she seemed to understand that we don't have a lot of time remaining. Technically, once we're able to associate embeddable mentee forms with the Organization (and create matches based on that) all minimum viable product functionality for the project will have been met. Therefore, anything else is considered as bonus. We should discuss as a team how we choose to proceed.
+
+Stylistic wise, the client doesn't have any preferences, since the organization taking over the application might refactor that completely. However, she does prefer Ant Design's notification component over the pop-ups.
+
+From our conversations with the client, the priority seems to be:
+
+- Finalizing the iFrame form so that it's associated with the organization.
+- Creating matches with Mentees and Mentors from common organizations.
+- Switching to the notification component from the pop-ups.
+
+Pace A then ran their application to show the new nav bar they created and the ability to only access the Auth page when logged in.
+
+Pace B demonstrated what they worked on, and they modified their admin and mentee forms, as well as the ability to send emails.
+
+### ACTION ITEMS
+
+2/24-01. If Codebar does decide to clone our repository to work on what we've done after April, does everyone on our team give their consent? A reminder that we signed the IP Agreement at the beginning of the course, and so I don't think we can object to this. However, the client asked if our team would be alright with Codebar forking our repo (if we're chosen out of the two groups), and so I want to make sure I have everyone's input before I agree or disagree. We should probably ask the instructor about this as well, since we signed for Pace, and I'm not sure if there's surrounding legalities of another organization using the project.
```

Comment:
I give my full consent to any partners of pace to use this repo in whatever way they want.